app/javascript/schools/courses/components/students_editor/StudentsEditor__TeamsList.res

```diff
@@ -4,6 +4,7 @@ let str = React.string
 
 open StudentsEditor__Types
 
+type displayCount = {displayCount: int, totalCount: int}
```

Comment:
Why do we need cache `displayCount` in a variable? You can always compute it by calling Array.length on the list.